### PR TITLE
event and resource docs first pass:

### PR DESCRIPTION
**Pull request by WillRaphaelson**
Adds conceptual and UI documentation for resources and events.

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.

### CODE REVIEW

File: docs/img/ui/event-feed.png
Comment (pleek91) [inline docs/img/ui/event-feed.png:1]: This is a mockup and doesn't reflect the actual application.

File: docs/concepts/events-and-resources.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+description: Events and resources are the primary units of observability in Prefect Cloud
+tags:
+    - resources
+    - events
+    - observability
+---
+
+# Events
+
+Events are the primary unit of observability in Prefect Cloud, and are designed to be a simple but complete record of something occuring somewhere in your stack at a given time. Prefect events can represent API calls, state changes, or events from your execution environment or infrastructure. Events enable both analysis of your data stack via (event feeds[link]) and the confiuration of stack reactivity via (automations[link])
```
Comment (pleek91) [inline docs/concepts/events-and-resources.md:11]: "Events and resources are the primary units of observability..."
"Events are the primary unit of observability..."

Should these be consistent?

File: docs/img/ui/event-feed.png
Comment (WillRaphaelson) [reply docs/img/ui/event-feed.png:1]: yep! just a placeholder

**Comment by WillRaphaelson** [comment]
This is ready for copy review, screenshots to be updated tomorrow.

File: docs/concepts/events-and-resources.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+description: Events are the primary units of observability in Prefect Cloud
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:2]:
```suggestion
description: An event is a notification of a change from a resource
```

File: docs/concepts/events-and-resources.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+tags:
+    - resources
+    - events
+    - observability
+---
+
+# Events
+
+Events are the primary unit of observability in Prefect Cloud, and are designed to be a simple but complete record of something occuring somewhere in your stack at a given time, and power several features in prefect cloud, including flow run logs, audit logs, and automations. Prefect events can represent API calls, state changes, or events from your execution environment or infrastructure. Events enable analysis of your data stack via [event feed](/ui/events/), and the confiuration of stack reactivity via [automations](/ui/automations/).
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:11]:
```suggestion
An event is a notification of a change. Together, events form a feed of activity recording what's happening across your stack. Events power several features in Prefect Cloud, including flow run logs, audit logs, and automations. Events can represent API calls, state transitions, or changes in your execution environment or infrastructure. Events enable observability into your data stack via [event feed](/ui/events/), and the configuration of Prefect's reactivity via [automations](/ui/automations/).
```

Comment (billpalombi) [reply docs/concepts/events-and-resources.md:11]: FWIW, I don't think is accurate to describe events as a "complete" record. I would prefer to reserve the word "analysis" for more synthetic representations of the information.

File: docs/concepts/events-and-resources.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## Event Specificiation
+
+Events adhere to a structured [specification](https://app.prefect.cloud/api/docs#tag/Events) for consistency.
+
+![Prefect UI](../img/ui/event-spec.png)
+  
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| occurred |  String | yes | When the event happened from the sender's perspective |
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:23]: Would it be simpler to say that it is the time that the event was sent? Is that accurate?

File: docs/concepts/events-and-resources.md (hunk 3 above)
Comment (cicdw) [inline docs/concepts/events-and-resources.md:11]: - capitalization on `prefect cloud`
- "via the event feed" (added "the")
- spelling of `configuration`

File: docs/concepts/events-and-resources.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+# Events
+
+Events are the primary unit of observability in Prefect Cloud, and are designed to be a simple but complete record of something occuring somewhere in your stack at a given time, and power several features in prefect cloud, including flow run logs, audit logs, and automations. Prefect events can represent API calls, state changes, or events from your execution environment or infrastructure. Events enable analysis of your data stack via [event feed](/ui/events/), and the confiuration of stack reactivity via [automations](/ui/automations/).
+
+![Prefect UI](../img/ui/event-feed.png)
+
+## Event Specificiation
+
+Events adhere to a structured [specification](https://app.prefect.cloud/api/docs#tag/Events) for consistency.
```
Comment (cicdw) [inline docs/concepts/events-and-resources.md:17]: i don't think you need to justify having a specification with "for consistency"

File: docs/concepts/events-and-resources.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+| resource|  Object | yes | The primary Resource this event concerns |
+| related | Array | no | A list of additional Resources involved in this event |
+| payload | Object | no | An open-ended set of data describing what happened |
+| id | String | yes | The client-provided identifier of this event |
+| follows | String | no | The ID of an event that is known to have occurred prior to this one. |
+
+
+## Event Grammar
+
+Generally, events have a consistent and informative grammar - an event describes a resource and an action that a resource took. For example, events emitted by Prefect objects take the form of:
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:34]:
```suggestion
Events have a consistent and informative grammar - an event describes a resource and an action that a resource took. For example, events emitted by Prefect objects take the form of:
```

File: docs/concepts/events-and-resources.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+```
+prefect.block.write-method.called
+prefect-cloud.automation.action.executed
+prefect-cloud.user.logged-in
+```
+
+## Event Sources
+
+Events are automatically emitted by all prefect objects, including flows, tasks, deployments, work queues, and logs. Prefect-emitted events will contain the `prefect` or `prefect-cloud` resource prefix. Events can also be sent to the prefect [events API](https://app.prefect.cloud/api/docs#tag/Events) via authenticated http request.
```
Comment (cicdw) [inline docs/concepts/events-and-resources.md:45]: prefect capitalization

**Review by cicdw — COMMENTED** [review]
minor copy edits

File: docs/concepts/events-and-resources.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+from prefect.events import emit_event
+
+def some_function(name: str="kiki") -> None:
+    print(f"hi {name}!")
+    emit_event(event=f"{name}.sent.event!", resource=f"prefect.resource.id":"coder.{name}")
+          
+some_function()
+```
+
+This event now appears in the [event feed](/ui/events/) where you can visualize activity in context and configure [automations](/ui/automations/) to react to the presence or absence of it in the future.
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:61]:
```suggestion
Emitted events will appear in the [event feed](/ui/events/) where you can visualize activity in context and configure [automations](/ui/automations/) to react to the presence or absence of it in the future.
```

File: docs/concepts/events-and-resources.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+Every event has a primary resource, which describes what object emitted an event. Resources are used as quasi-stable identifiers for sources of events, and are consutrcted as dot-delimited strings such as:
+
+```
+prefect-cloud.automation.5b9c5c3d-6ca0-48d0-8331-79f4b65385b3.action.0
+acme.user.kiki.elt_script_1
+prefect.flow-run.e3755d32-cec5-42ca-9bcd-af236e308ba6
+```
+
+Resources can optionally have additional arbitrary labels which can be used in different event aggregatino queries, such as:
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:74]:
```suggestion
Resources can optionally have additional arbitrary labels which can be used in event aggregation queries, such as:
```

File: docs/ui/events.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    - UI
+    - dashboard
+    - Prefect Cloud
+    - Observability
+    - Events
+---
+
+# Events
+
+Prefect Cloud provides an interactive dashboard to analyze and take action on events that occurred in your worksapce on the [event feed page](/concepts/events-and-resources/).
```
Comment (billpalombi) [inline docs/ui/events.md:14]:
```suggestion
Prefect Cloud provides an interactive dashboard to analyze and take action on events that occurred in your workspace on the [event feed page](/concepts/events-and-resources/).
```

File: docs/ui/events.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+![Event feed](../img/ui/event-feed.png)
+
+## Events feed
+
+The event feed is the primary place to view, search, and filter events to understand how data is moving through your stack. Each entry displays data on the resource, related resource, and event that took place.
```
Comment (billpalombi) [inline docs/ui/events.md:20]:
```suggestion
The event feed is the primary place to view, search, and filter events to understand activity across your stack. Each entry displays data on the resource, related resource, and event that took place.
```

File: docs/ui/events.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+![Event detail](../img/ui/event-detail.png)
+
+
+## Automating based on events
+
+From an event page, you can easily configure an automation to trigger on the observation of matching events or a lack of matching events by clicking the automate button in the overflow menu:
+
+![Automation from event](../img/ui/automation-from-event.png)
+
+The default trigger configurtion will fire every time it sees an event with a matching resource identifier. Advanced confiuguration is possible via [custom triggers](/ui/automations/).
```
Comment (billpalombi) [inline docs/ui/events.md:35]:
```suggestion
The default trigger configuration will fire every time it sees an event with a matching resource identifier. Advanced configuration is possible via [custom triggers](/ui/automations/). 
```

**Review by billpalombi — COMMENTED** [review]
First pass. Mostly copyediting. Will continue to think about how to best frame this.

File: docs/concepts/events-and-resources.md [hunk 13]
```diff
@@ -0,0 +1,104 @@
...
+
+## Event Specificiation
+
+Events adhere to a structured [specification](https://app.prefect.cloud/api/docs#tag/Events).
+
+![Prefect UI](../img/ui/event-spec.png)
+  
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| occurred |  String | yes | When the event happened from the sender's perspective |
```
Comment (WillRaphaelson) [inline docs/concepts/events-and-resources.md:23]:
```suggestion
| occurred |  String | yes | When the event happened |
```

File: docs/ui/events.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+The event feed is the primary place to view, search, and filter events to understand activity across your stack. Each entry displays data on the resource, related resource, and event that took place.
+
+## Event details
+
+You can view more information about an event by clicking into it, where you can view the full details of an events resource, related resources, and payload.
+
+![Event detail](../img/ui/event-detail.png)
+
+
+## Automating based on events
```
Comment (billpalombi) [inline docs/ui/events.md:29]:
```suggestion
## Reacting to events
```

File: docs/concepts/events-and-resources.md [hunk 15]
```diff
@@ -0,0 +1,104 @@
...
+| resource|  Object | yes | The primary Resource this event concerns |
+| related | Array | no | A list of additional Resources involved in this event |
+| payload | Object | no | An open-ended set of data describing what happened |
+| id | String | yes | The client-provided identifier of this event |
+| follows | String | no | The ID of an event that is known to have occurred prior to this one. |
+
+
+## Event Grammar
+
+Generally, events have a consistent and informative grammar - an event describes a resource and an action that a resource took or that was taken on that resource. For example, events emitted by Prefect objects take the form of:
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:34]:
```suggestion
Generally, events have a consistent and informative grammar - an event describes a resource and an action that the resource took or that was taken on that resource. For example, events emitted by Prefect objects take the form of:
```

File: docs/concepts/events-and-resources.md [hunk 16]
```diff
@@ -0,0 +1,104 @@
...
+prefect.block.write-method.called
+prefect-cloud.automation.action.executed
+prefect-cloud.user.logged-in
+```
+
+## Event Sources
+
+Events are automatically emitted by all Prefect objects, including flows, tasks, deployments, work queues, and logs. Prefect-emitted events will contain the `prefect` or `prefect-cloud` resource prefix. Events can also be sent to the Prefect [events API](https://app.prefect.cloud/api/docs#tag/Events) via authenticated http request.
+
+Prefect provides an events client that emits events, for use in arbitrary python code that may not be a task or
```
Comment (billpalombi) [inline docs/concepts/events-and-resources.md:47]:
```suggestion
The Prefect SDK provides a method that emits events, for use in arbitrary python code that may not be a task or 
```

File: docs/ui/events.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+
+![Event feed](../img/ui/event-feed.png)
+
+## Event feed <span class="badge beta"></span>
+
+The event feed is the primary place to view, search, and filter events to understand activity across your stack. Each entry displays data on the resource, related resource, and event that took place.
+
+## Event details
+
+You can view more information about an event by clicking into it, where you can view the full details of an events resource, related resources, and payload.
```
Comment (billpalombi) [inline docs/ui/events.md:24]:
```suggestion
You can view more information about an event by clicking into it, where you can view the full details of an event's resource, related resources, and its payload.
```

**Review by billpalombi — APPROVED** [review]
Some copyedits but otherwise lgtm

**Pull request merged**